module/char/charpick.c

```diff
@@ -110,6 +110,21 @@ void ChooseCharacter(Character *characters, WORD num_characters, char *motd,
    }
 }
 /********************************************************************/
+/*
+ * compareCharacters:  Sorting alphabetically with new character slots always at the end.
+ */
+bool compareCharacters(const Character& a, const Character& b) {
+    // Place characters with flags != 1 (new users) at the end
+    if (a.flags != 1 && b.flags == 1) {
+        return true;
+    }
+    if (a.flags == 1 && b.flags != 1) {
+        return false;
+    }
+    // If flags are the same and both have flags == 1, compare names case-insensitively

```

Comment:
I think this comment isn't right.  We reach here if the flags are the same, not if they're 1.